src/index.js

```diff
@@ -64,24 +65,96 @@ class PubsubBaseProtocol extends EventEmitter {
     this.topics = new Map()
 
     /**
-     * Cache of seen messages
+     * Map of peers.
      *
-     * @type {TimeCache}
+     * @type {Map<string, Peer>}
      */
-    this.seenCache = new TimeCache()
```

Comment:
Removed and added in the implementations. I think that we should not assume that all pubsub implementations will use it